languages/csharp/exercises/concept/switch-statements/.meta/Example.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+
+// **** please do not modify the Manager class ****
+public class Manager
+{
+    public string Name { get; }
+    public string Activity { get; }
+
+    public Manager(string name, string activity)
+    {
+        this.Name = name;
+        this.Activity = activity;
+    }
+}
+
+// **** please do not modify the Incident enum ****
+public enum Incident
+{
+    RedCard,
+    YellowCard,
+    Fowl,
```

Comment:
Should this be `Foul`?

```suggestion
    Foul,
```